emsdk.py

```diff
@@ -1695,12 +1688,12 @@ def is_installed_version(self):
     version_file_path = self.get_version_file_path()
     if os.path.isfile(version_file_path):
       with open(version_file_path, 'r') as version_file:
-        return version_file.read() == self.name
+        return version_file.read().strip() == self.name
     return False
 
   def update_installed_version(self):
     with open(self.get_version_file_path(), 'w') as version_file:
-      version_file.write(self.name)
+      version_file.write(self.name + '\n')
```

Comment:
why add this to remove it elsewhere?

Reply:
Just a drive-by cleanup.  Since its text file it should always end in newline (i noticed because I run `cat <version_file>` which looks odd without the ending newline).

Reply:
I'm not opposed but maybe it makes this technically not NFC? sgtm either way.